src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,118 @@
 package com.theironyard.novauc;
+import java.util.Scanner;
 
+import java.util.HashMap;
 public class Main {
+    public static Scanner scanner = new Scanner(System.in);
+    public static Customer customer = new Customer();
+    public static HashMap<String, Double> people = new HashMap<String, Double>();
 
-    public static void main(String[] args) {
-	// write your code here
+
+    public static void main(String[] args) throws Exception {
+        // write your code here
+
+
+
+            people.put("Rodney", 1200.00);
+            people.put("Bradley", 1500.00);
+            people.put("George", 2750.50);
+            people.put("William", 2200.12);
+            people.put("Ronald", 3515.25);
+            people.put("Hallie", 5500.00);
+
+        boolean run = true;
+        while (run) {
+
+            System.out.print("Hello,");
+
+            customer.chooseName();
+            //   customer.chooseAccount();
+            customer.chooseTransaction();
+
+//
+//        customer.findType("1. Display Balance");
+//        customer.findType("2. Withdrawal");
+//        customer.findType("3. Deposit");
+//        customer.findType("4. Remove Account");
+//        customer.findType("5. Cancel");
+
+  //          String line = scanner.nextLine();
+
+            {
+
+
+            }
+
```

Comment:
Well done Jacob.  In the future I would clean things up a bit.  Example brackets on line 41 and 44 that don't need to be there.  A couple of variables you never use etc.  But that is a minor thing.  The project again is really well done.